inc/dbconnection.class.php

```diff
@@ -300,6 +312,7 @@ static function establishDBConnection($use_slave, $required, $display = true) {
                   $res = self::switchToSlave();
                }
                if ($res) {
+                   //FIXME: no longer possible
```

Comment:
This should be fixed before merging the PR. Maybe it can be dropped (see previous comment).

Reply:
Yep, I've dropped